provider/resources.go

```diff
@@ -137,7 +140,12 @@ func Provider() tfbridge.ProviderInfo {
 			"digitalocean_domain": {
 				Tok: makeResource(digitalOceanMod, "Domain"),
 				Fields: map[string]*tfbridge.SchemaInfo{
-					"name": {Name: "name"},
+					"name": {
+						Name: "name",
+					},
+					"urn": {
+						Name: "DomainUrn",
```

Comment:
These should all be `camelCase`.

```suggestion
						Name: "domainUrn",
```

Reply:
sorry, fixed!!